processing/src/main/java/io/druid/query/metadata/metadata/SegmentAnalysis.java

```diff
@@ -140,7 +140,7 @@ public int hashCode()
     result = 31 * result + (interval != null ? interval.hashCode() : 0);
     result = 31 * result + (columns != null ? columns.hashCode() : 0);
     result = 31 * result + (int) (size ^ (size >>> 32));
-    result = 31 * result + (int) (numRows ^ (numRows >>> 32));
+    result = 31 * result + numRows;
```

Comment:
`numRows` is int value and `(int) (numRows ^ (numRows >>> 32))` is same with `numRows`, I think.